frontend/src/exporter/ExportedInsight/ExportedInsight.tsx

```diff
@@ -11,8 +11,8 @@ import { ExportOptions, ExportType } from '~/exporter/types'
 import clsx from 'clsx'
 import { SINGLE_SERIES_DISPLAY_TYPES } from 'lib/constants'
 import { isTrendsFilter } from 'scenes/insights/sharedUtils'
-import { Query } from '~/queries/Query/Query'
 import { isDataTableNode, isInsightVizNode } from '~/queries/utils'
+import QueriesUnsupportedHere from 'lib/components/Cards/InsightCard/QueriesUnsupportedHere'
```

Comment:
Late yolo nit: let's [avoid default exports](https://posthog.com/handbook/engineering/conventions/frontend-coding) when possible.

Reply:
👍 is it only a consistency thing or ...?

Reply:
It also helps bundlers and IDEs, since default exports by definition lose their name when exported. Some things just work... smoother/slicker this way 🤷

Reply:
sorted in #14588